Wordpress/wordpress-plugin-installation.md

```diff
@@ -0,0 +1,82 @@
+{{{
+  "title": "WordPress Plugin Installation",
+  "date": "07-15-2015",
+  "author": "Bill Burge",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### IMPORTANT NOTE

CenturyLink Cloud WordPress hosting is currently in a Limited Beta program with specific customers by invitation only and is not intended for production usage.

During the Limited Beta there is no production Service Level Aggreement.

--
+
+CenturyLink Cloud WordPress sites use Git to deploy new WordPress code including new plugins.
+
+In order to activate a plugin you will need to download a WordPress Plugin and upload it into the Master branch of the Centurylink Git repository provided to you during your CenturyLink Cloud Wordpress site creation. This will then force a refresh of your WordPress blog and allow you to activate and configure the plugin.
+
+**NOTE:** These instructions assume the following...
+
+* The Git CLI is installed
+* A working knowledge of Git usage
+
+##Deploying a WordPress Plugin Using the Git CLI
+
+1. [Clone your WordPress Git Repo](wordpress-clone-push-gitlab.md "Clone your WordPress Git Repo")
+
+2. Open the wp-content folder.
+
+  ![](../images/wp_plugin_installation/wp_plugin_installation_01.png "wp_plugin_installation_01.png")
+
+3. Open the plugins folder.
+
+  ![](../images/wp_plugin_installation/wp_plugin_installation_02.png "wp_plugin_installation_02.png")
+
+4. Download the WordPress Plugin you want to install.
+
+  _In this example we will be installing Akismet_.
+
+5. Locate the local folder the plugin was downloaded to and copy it into the plugins folder of your cloned GitHub repository.
+
+  ![](../images/wp_plugin_installation/wp_plugin_installation_03.png "wp_plugin_installation_03.png")
+
+  You will end up with the following...
+
+  ![](../images/wp_plugin_installation/wp_plugin_installation_04.png "wp_plugin_installation_04.png")
+
+6. Inside your repo, at the command line, running **_git status_** will now show you uncommited changes.
+
+  ![](../images/wp_plugin_installation/wp_plugin_installation_05.png "wp_plugin_installation_05.png")
+
+7. Commit your changes using the **_git add_** command.
+
+  _In this example we will run git add * to add all files to the repo_
```

Comment:
use code block for input not bold...

`code block` vs. **bold**

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code